README.md

```diff
@@ -127,6 +127,10 @@ via Conda or Docker.
 ## Build/Install from Source
 See the build [guide](BUILD.md).
 
+## Scikit-learn Compatibility
+
+cuML is compatible with scikit-learn version 1.5 or higher.
```

Comment:
"or higher" reads odd to me, but maybe it's just Monday morning. "and higher"? "and up"? Maybe just a version constraint? Feel free to ignore, maybe I'm just overthinking.

```suggestion
cuML is compatible with scikit-learn >= 1.5.
```